js/google_ima.js

```diff
@@ -1818,6 +1838,16 @@ require("../html5-common/js/utils/utils.js");
           case eventType.USER_CLOSE:
           case eventType.SKIPPED:
           case eventType.COMPLETE:
+            //Workaround of an issue on iOS where the IMA iframe is capturing clicks after an ad.
+            //We will show the iframe on receiving STARTED and hide it when receiving COMPLETE
+            if (OO.isIos) {
```

Comment:
do we want to check isIos or do we want to check for single element video?  Otherwise this effects both iphone and ipad.  But is the issue occuring on ipad?

Reply:
I haven't tested iPad. This change is unrelated to single video element mode.

Reply:
This issue is observed on iPad as well.

Reply:
ok, you can disregard this then.